### PR TITLE
fix: bump contrast on attachment overlay text

### PR DESCRIPTION
**Pull request by KeeJef**
This is a proxy pull request from the following codeberg repo, https://codeberg.org/gravel/session-desktop/commits/branch/more-images-count-visibility

```
When viewing a message with too many image attachments,
a preview of the remaining images is shown,
overlaid with an indicator of the remaining image count.

The indicator was, however, poorly visible on light image backgrounds.

This commit increases the text size of the indicator and includes
a drop shadow on the indicator text, thus improving the indicator's
readability.

```
Relates: to oxen-io/session-desktop-temp#213

### CODE REVIEW

**Comment by Bilb** [comment]
Would be good to have a screenshot of how it looks after the change, but I'll give it a try locally.

**Comment by Bilb** [comment]
image
I am not sure this is going to be enough.
Can you force the background to be darker with something like this instead on the `.module-image__text-container {`
...  background-color: rgba(0, 0, 0, 0.4);

**Comment by Bilb** [comment]
will check with design team

File: stylesheets/_modules.scss [hunk 1]
```diff
@@ -834,11 +834,13 @@
...
   color: var(--white-color);
 
-  font-size: 20px;
+  font-size: 30px;
   font-weight: normal;
   letter-spacing: 0;
 
   text-align: center;
+  --text-shadow-offset: 0.05em;
+  text-shadow: var(--black-color) var(--text-shadow-offset) var(--text-shadow-offset) 0.15em;
```
Comment (Bilb) [inline stylesheets/_modules.scss:843]:
```suggestion
  font-size: 20px;  
  font-weight: normal;
  letter-spacing: 0;
  text-align: center;
  background-color: rgba(0, 0, 0, 0.7);

```

**Comment by Bilb** [comment]
@KeeJef design said we want 
`  background-color: rgba(0, 0, 0, 0.7);`
no text-shadow 
and font size back to 20xp. Can I let you update this PR or I make one on the side?

**Comment by KeeJef** [comment]
I can update

**Comment by KeeJef** [comment]
Updated

**Comment by Bilb** [comment]
lgtm

**Review by Bilb — APPROVED** [review]

**Pull request merged**